apollo-rxsupport/build.gradle

```diff
@@ -0,0 +1,61 @@
+buildscript {
+  dependencies {
+    classpath dep.androidPlugin
+  }
+}
+
+apply plugin: 'com.android.library'
```

Comment:
it depends on Runtime which is currently an android lib.  If/when we move sql stuff out to its own module we can change this to java